lib/releases.js

```diff
@@ -79,12 +80,20 @@ const contributorsSentence = ({ commits, pullRequests }) => {
 const getCategoriesConfig = ({ config }) => {
   const categoriesConfig = config.categories.reduce((acc, category) => {
     acc[category.label] = category
+    if (category.labels && category.labels.length) {
+      category.labels.map(label => (acc[label] = category))
+    }
     return acc
   }, {})
 
   const orderedCategories = [
     UNCATEGORIZED,
-    ...config.categories.map(category => category.label)
+    ..._.flatMap(config.categories, category => {
+      if (category.label) return category.label
```

Comment:
Would it make sense to use a ternary here? I always find them to be more expressive than `if/else` :)

Reply:
No, cause both can be undefined if you would so like 😢 I prefer the more expressive approach in this case

Reply:
But it's fine if both are `undefined` right? Then it will just return `undefined`, just like it does now. Or am I misunderstanding?

Reply:
You are right 🤦‍♂